public/jsmodules/querycity.js

```diff
@@ -0,0 +1,71 @@
+let queryCity = (xhr, textInput, countryInput, searchList) => {
```

Comment:
Don't pass the `XHR` object as a variable. This query city method should create a new object every time